transitions/registration.js

```diff
@@ -265,6 +265,7 @@ module.exports = {
             if (err) {
                 callback(err);
             } else if (registrations.length) { // id collision, retry
+                logger.warn('Registration ID ' + id + ' is not unique, retrying...');
```

Comment:
Fair enough. It'd be nice to notify admins when the ID space is getting close to full, but that's a problem for another day.